onboarding-extension-requests/README.md

```
@@ -0,0 +1,100 @@
+| oldEndsOn     | number    | Old deadline of the onboarding task in millisecond                      |
+| newEndsOn     | number    | New deadline of the onboarding task in millisecond                      |
+| message       | String    | The message provided by super user while approving or rejecting request.| 
+| createdAt     | Timestamp | Unix timestamp for the creation time of the request.                    |
+| updatedAt     | Timestamp | Unix timestamp for the last update time of the request.                 |
+| lastModifiedBy| String    | The id of the superuser or request owner who processed the request      |  
+| reason        | String    | The reason for the extension request.                                   |
+| requestedBy   | String    | The username of the user who created the request.                       |
+| requestNumber | number    | The current request number for onboarding task.                         |
+| type          | String    | The type of request i.e. ONBOARDING.                                    |
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)